src/libraries/Descriptor.sol

```diff
@@ -503,6 +503,10 @@ library Descriptor {
         }
     }
 
+    function isSpecialCharacter(bytes1 b) private pure returns (bool) {
+        return b == '"' || b == "\u000c" || b == "\n" || b == "\r" || b == "\t";
```

Comment:
am i mad? surely `\u000c` is longer than a single byte?

Reply:
I had exactly the same thought! Turns out it get's properly escaped into 1 bytes:

```
pragma solidity ^0.8.0;

contract RandomTest {
    function stringToBytes32() public pure returns (uint256 length) {
        string memory source = "\u000c";
        bytes memory tempEmptyStringTest = bytes(source);
        return tempEmptyStringTest.length;
    }
}
```

If you test this out in Remix the return value is 1 :)

Reply:
wild, thanks for the poc :)